test/integration/scheduler_perf/config/performance-config.yaml

```diff
@@ -688,6 +688,49 @@
 #      initPods: 20000
 #      measurePods: 5000
 
+# Measure throughput of regular schedulable pods that are interleaved by high priority preemption pods scheduled at 20/s rate.
+# Implementation of asynchrounous preemption feature https://github.com/kubernetes/kubernetes/issues/126858 is supposed to increase the trhoughput of the measured pods as all heavy operations (apiserver communication) will be performed asyncronously, wihtout blocking the scheduler loop.
+# How is it achieved:
+# 1. There are X initial nodes with 4 low priority pods each, consuming 3.6 CPU out of the total 4 available.
+# 2. High priority preemption which need to preempt 3 of 4 low priority pods to fit (require 3 CPU).
```

Comment:
Maybe we should make a preemptor Pod smaller? so that more preemption can be triggered in the scenario.

Currently, the number of triggered preemption equals the number of nodes. 
So, (at the largest scenario) if `churn` finishes creating 5000 preemptor Pods before the last `createPods` finishes 5000th Pod creation, some Pods created later won't get impacted by the preemption.

Reply:
We trigger preemptions at 20/s rate, so the test would have to last 250s to run into this problem.

In general, I was thinking about using the `number` param (after parametrizing it) to define a limit for churn, but in fact reaching the limit during the test should fail it. If that happens, then the test should be rebalanced. In this case we should have quite big safe margin.

Reply:
Ah right, I just overlooked the specified interval.